web/index.html

```diff
@@ -97,6 +132,7 @@
             ferrostar.startNavigation(route, config);
           },
         });
+        mapInstance.addControl(searchBox, "top-left");
 
         ferrostar.center = { lng: -122.42, lat: 37.81 };
         ferrostar.zoom = 18;
```

Comment:
GitHub is stupid and doesn't let me comment on lines below this, but the following line should be removed now:

```javascript
        ferrostar.customStyles = searchBoxStyle;
```

Pending the result of the other large discussion on `ferrostar-map.ts`, we should also delete the `configureMap` assignment, since this is currently broken and should probably be deleted.